### PR TITLE
SONARKT-69 Rule S6300: Using unencrypted files in mobile applications is security-sensitive

### PR DESCRIPTION
**Pull request by margarita-nedzelska-sonarsource**

### CODE REVIEW

**Review by johann-beleites-sonarsource — APPROVED** [review]
Some smaller points, then LGTM!

File: sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/AbstractCheck.kt [hunk 1]
```diff
@@ -75,6 +75,8 @@ abstract class AbstractCheck : KotlinCheck, KtVisitor<Unit, KotlinFileContext>()
         secondaryLocations: List<SecondaryLocation> = emptyList(),
         gap: Double? = null,
     ) = reportIssue(textRange(psiElement), message, secondaryLocations, gap)
+    
+    internal fun KotlinFileContext.isInAndroid() = inputFileContext.isAndroid
```
[the remark this answers is not in the document]
Comment (margarita-nedzelska-sonarsource) [reply sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/AbstractCheck.kt:79]: I think it just required a little less code. Anyway after moving away from slang, it seems to me we can have only one file context, could be another ticket for the future refactoring :) wdyt?

[the remark this answers is not in the document]
Comment (johann-beleites-sonarsource) [reply sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/AbstractCheck.kt:79]: Yea, I agree :)

[the remark this answers is not in the document]
Comment (margarita-nedzelska-sonarsource) [reply sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/AbstractCheck.kt:79]: Here is [the ticket](https://jira.sonarsource.com/browse/SONARKT-113) :)

**Pull request merged**